moa/src/main/java/moa/classifiers/meta/StreamingRandomPatches.java

```diff
@@ -76,7 +76,7 @@
     private static final long serialVersionUID = 1L;
 
     public ClassOption baseLearnerOption = new ClassOption("baseLearner", 'l',
-            "Classifier to train on instances.", Classifier.class, "trees.HoeffdingTree -g 50 -c 0.01");
+            "Classifier to train on instances.", Classifier.class, "trees.ARFFIMTDD -g 50 -c 0.01");
```

Comment:
The default base learner for StreamingRandomPatches **should not** be ARFFIMTDD.